docs/readme.md

```diff
@@ -0,0 +1,25 @@
+# Taxonomy
+
+## Unclassified Tree
+
+Nodes are deemed *unclassified* when their *scientific name* contains (ignore case) the word `unclassified`; these generate the **Unclassified** tree.
+
+## Environmental Samples Tree
+
+The **Environmental Samples** tree is generated by the nodes whose *scientific name* has prefix (ignore case) `environmental samples`.
+
+# File structure
+
+Each relation is stored as two `csv` files (in and out): first column source, second column target. We have the following relations
+
+1. `parent` (the tree structure)
+2. `scientific_name`
+3. `rank`
+
+For all these entities we use their obvious `String` representation.
+
+Of course, we have one set of this files for each of the trees we provide (full, good, unclassified, environmental samples).
```

Comment:
How do we define the *good tree*? The full tree minus the (*unclassified* or *environmental samples*) nodes branches? Should we keep the ancestors of the *bad* nodes in the *good tree*?

(*The good tree* sounds like something biblical :angel:)